eoc_journal/translations/en/LC_MESSAGES/text.po-e

```diff
@@ -0,0 +1,247 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
```

Comment:
Is this file required? Looks like a copy of the above.

Reply:
@viadanna That's a backup file created by sed. I'll add it to `.gitignore`